code/game/objects/items/devices/nyamagotchi.dm

```diff
@@ -0,0 +1,182 @@
+/obj/item/nyamagotchi
+	name = "nyamagotchi"
+	icon = 'icons/obj/devices/nyamagotchi.dmi'
+	desc = "A small electronic 'pet' that requires care and attention. An ancient relic sure to evoke nostalgic feelings."
+	icon_state = "default"
+	worn_icon_state = "nothing"
+	base_icon_state = "default"
+	slot_flags = ITEM_SLOT_BELT
+	w_class = WEIGHT_CLASS_SMALL
+	force = 2
+	throwforce = 2
+	lefthand_file = 'icons/mob/inhands/items/devices_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/items/devices_righthand.dmi'
+	inhand_icon_state = "electronic"
+	custom_materials = list(/datum/material/iron=SMALL_MATERIAL_AMOUNT, /datum/material/glass=SMALL_MATERIAL_AMOUNT, /datum/material/plastic=SMALL_MATERIAL_AMOUNT)
+	// interaction_flags_atom = parent_type::interaction_flags_atom | INTERACT_ATOM_ALLOW_USER_LOCATION | INTERACT_ATOM_IGNORE_MOBILITY
+	var/list/icons_available = list()
+	var/radial_icon_file = 'icons/hud/radial_nyamagotchi.dmi'
+
+	var/hunger = 0      // Hunger level (0 = full, 100 = starving)
```

Comment:
```suggestion
	/// Hunger level (0 = full, 100 = starving)
	var/hunger = 0
```
Just as an FYI for the future. we use three when defining variables so that autodocs can generate documentation.

.github/guides/AUTODOC.md has full information on how it works.